nixos/modules/i18n/inputMethod/default.xml

```diff
@@ -0,0 +1,99 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="module-services-input-methods">
+
+<title>Input Methods</title>
+
+<para>Input methods are an operating system component that allows any data, such as keyboard strokes or mouse movements, to be received as input. In this way users can enter characters and symbols not found on their input devices. Using an input method is obligatory for any language that has more graphemes than there are keys on the keyboard.</para>
+
+<para>The following input methods are available in NixOS:</para>
+
+<itemizedlist>
+  <listitem><para>IBus: The intelligent input bus.</para></listitem>
+  <listitem><para>Fcitx: A customizable lightweight input method.</para></listitem>
+  <listitem><para>Nabi: A Korean input method based on XIM.</para></listitem>
+  <listitem><para>Uim: The universal input method, is a library with a XIM bridge.</para></listitem>
+</itemizedlist>
```

Comment:
I don't really know what this means (in fact I've never heard of these input methods). What about (say) `evdev`